README.md

```diff
@@ -2,13 +2,16 @@
 
 An [apcaccess](https://github.com/mapero/apcaccess) wrapper for [Homebridge](https://github.com/nfarina/homebridge). Supports precisely one UPS. If you're on battery power, the Contact Sensor is open.
 
+Optional temperature sensor added by [ThisIsQasim](https://github.com/ThisIsQasim). Logging improvements by [davidjbradshaw](https://github.com/davidjbradshaw). Thank you!
```

Comment:
I'd move this to the change log and just put the name next to the feature. Contributors also appear in the right sidebar on the main project page.

This is how I do it on a project of mine.

https://github.com/davidjbradshaw/iframe-resizer/blob/master/CHANGELOG.md